packages/performance/src/services/perf_logger.test.ts

```diff
@@ -108,12 +113,136 @@ describe('Performance Monitoring > perf_logger', () => {
 
     it('does not log an event if cookies are disabled in the browser', () => {
       stub(Api.prototype, 'requiredApisAvailable').returns(false);
+      stub(attributeUtils, 'getVisibilityState').returns(VISIBILITY_STATE);
       const trace = new Trace(TRACE_NAME);
       trace.record(START_TIME, DURATION);
       clock.tick(1);
 
       expect(addToQueueStub).not.to.be.called;
     });
+
+    it('Amount of customMetric is 32.', () => {
```

Comment:
Should be written as a third person sentence. Does not need a period at the end.

Reply:
Changed it to align with the rest of the test names.